### PR TITLE
Automatically update the java version

### PR DESCRIPTION
**Pull request by laura-ding**
- When release version, the script will uniformly modify the version of java.
- You can change the java version by this

```
cmake -DNEBULA_BUILD_VERSION=${version}  ${other_args} ..
```

### CODE REVIEW

File: etc/nebula-metad.conf.default [hunk 1]
```diff
@@ -15,6 +15,9 @@
 --logbufsecs=0
 
 ########## networking ##########
+# It is a list of IPs split by comma, the ips number equals replica number. If empty, it means replica is 1
+# The format looks like ip1, ip2, ip3
```
Comment (yixinglu) [inline etc/nebula-metad.conf.default]: ip:port

Comment (laura-ding) [reply etc/nebula-metad.conf.default]: good catch

**Comment by yixinglu** [comment]
reference #844

**Comment by laura-ding** [comment]
jenkins go

File: etc/nebula-metad.conf.default [hunk 2]
```diff
@@ -15,6 +15,8 @@
 --logbufsecs=0
 
 ########## networking ##########
+# List of meta server addresses, the format looks like ip1:port1, ip2:port2, ip3:port3
```
Comment (zhangguoqing) [inline etc/nebula-metad.conf.default]: It's good to add the 'meta_server_addrs' configure item to the nebula-metad.conf.

**Comment by yixinglu** [comment]
`meta_server_addrs` option has been added to `nebula-meta.conf` in #1111

**Pull request closed without merging**